Update releases-and-support.md (#37470)
dotnet/docs: docs/core/releases-and-support.md

---
# Releases and support for .NET

 > [!NOTE]

>  > This topic concerns .NET 5 and later versions, and versions of .NET Core, which preceded them. For .NET Framework releases, see [.NET Framework Lifecycle](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework).

Reviewer: ```suggestion
 > This article concerns .NET 5 and later versions, and versions of .NET Core. For .NET Framework releases, see [.NET Framework Lifecycle](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework).
```

Reviewer: I added something similar in https://github.com/dotnet/docs/pull/37459/files#diff-ca0bc46f959dcebe0ffc031205a1cf11dc60771902da75a4375f0d24e9317323, so I think this is great.

* .NET Core 3.1 is the first minor release after the .NET Core 3.0 major release.
* .NET Core 5.0.15 is the fifteenth patch for .NET 5.

> For a list of released versions of .NET and .NET Core, and information about how often .NET ships, see the [Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core#lifecycle).

Reviewer: ```suggestion
For a list of released versions of .NET and information about how often .NET ships, see the [Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core#lifecycle).
```
